test/functional/sqlsrv/MsHelper.inc

```diff
@@ -325,13 +326,26 @@ function getSeqPlaceholders($num)
 }
 
 /**
- * @return bool false if $keystore specified in MsSetup.inc is none or data not encrypted,
- *              otherwise return true
+ * @return bool false if $keystore specified in MsSetup.inc is none; return true otherwise
  */
 function isColEncrypted()
 {
     global $keystore, $dataEncrypted;
-    if ($keystore == KEYSTORE_NONE || !$dataEncrypted) {
+    if ($keystore === KEYSTORE_NONE) {
```

Comment:
If you change this to just check if the connection is column encryption enabled, what happens to all the tests that uses this function?

Reply:
Most tests only care to distinguish whether it's connected with ColumnEncryption enabled or not. Whether data is encrypted or not only affects whether the table columns are encrypted or not and some AE specific tests

Reply:
OK, as long as you've double check the usage of this function makes sense for each test